R/calc_views.R

```diff
@@ -61,6 +62,13 @@ group_views <- function(gsplot){
   return(append(views, non.views))
 }
 
+append_replace <- function(old.list, new.list){
+  out.list <- old.list
+  out.list[names(old.list) %in% names(new.list)] <- new.list[names(out.list[names(old.list) %in% names(new.list)])]
+  new.list[names(new.list) %in% names(old.list)] <- NULL
+  out.list <- append(out.list, new.list)
+  return(out.list)
+}
```

Comment:
working on that now...it still does a few important things, but I am trying to figure out what they are.